### DISCUSSION TITLE
Cool project but..

### DISCUSSION THREAD

**Discussion by Jarrodsz** [docker-mailserver/docker-mailserver#3185] [Q&A]
Looked at this from 2018
Now i always bought hosted mails for the hassle but since i have a growing number of mailboxes for projects and my hoster has a webmail where potentially everyone could bruteforce my mailbox 🤣 i wanted to have more secure centralised mail hosting.

one would think if dns rules like dmark etc are setup correctly mail delivery should not be much problem but then still so many reports of issues non recieved mails. 

In general for self hosted mails. A mid level hoster seems to do a better job delivering rock solid emails and always wondered why that is.

Can i use the project solid hosting of mailboxes without spending hours on debugging?

i also want my emails to myself so that nobody can “seize “ my mailcontents.

So very mich appreciate all the work just curious if this is stable enough?

If not would there be a good alternative?

**Comment by ontheair81** [comment]
I am using docker mailserver since around 1.5 years with multiple accounts / mailboxes and multiple domains.

My personal experience is, that everything works without any problems. Of course, setting everything up in the beginning was a little bit challenging - not because of the project itself, but due to the fact that it was my first time ever setting up an mailserver.

But as you already said: If DNS entries are correct, there usually should not be a big problem. And of course, you should use an ip on the server that is not on any blacklist.
I had a little problem with that in the beginning (Microsoft rejecting my mails, because there was a spammer in the same ip _range_, so they blocked the whole range including my ip), but this was solved by contacting them.

My suggestion is: Just try it and start with a domain and user for testing purposes only. If everything works as expected, move all your other domains and accounts there.

**Comment by Jarrodsz** [comment]
Ah great to hear. Thank you.
I will test it then fiy making the switch after years of fowwing this repro.

my post was niot to bias really appreciate all the effort that went into this
